Source/WebCore/dom/Document.cpp

```diff
@@ -27,7 +27,6 @@
 
 #include "config.h"
 #include "Document.h"
-
```

Comment:
This change should be undone.  WebKit style has config, the header with the same name as the cpp/mm, then a newline then other headers.

Reply:
ahh. Undid the change.